dev/Expander/Expander.xaml

```diff
@@ -22,36 +22,85 @@
+        <Setter Property="MinWidth" Value="{ThemeResource FlyoutThemeMinWidth}" />
+        <Setter Property="MaxWidth" Value="{ThemeResource FlyoutThemeMaxWidth}" />
+        <Setter Property="MinHeight" Value="{StaticResource ExpanderMinHeight}" />
+        <Setter Property="BorderThickness" Value="{ThemeResource ExpanderBorderThickness}" />
+        <Setter Property="BorderBrush" Value="{ThemeResource ExpanderBorderBrush}" />
+        <Setter Property="Padding" Value="{StaticResource ExpanderHeaderPadding}" />
         <Setter Property="HorizontalAlignment" Value="Stretch" />
-        <Setter Property="HorizontalContentAlignment" Value="Left"/>
-        <Setter Property="VerticalContentAlignment" Value="Center"/>
+        <Setter Property="HorizontalContentAlignment" Value="Left" />
+        <Setter Property="VerticalContentAlignment" Value="Center" />
         <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="local:Expander">
                     <Grid MinWidth="{TemplateBinding MinWidth}" MaxWidth="{TemplateBinding MaxWidth}">
                         <VisualStateManager.VisualStateGroups>
                             <VisualStateGroup x:Name="ExpandStates">
-                                <VisualState x:Name="Expanded">
+                                <VisualState x:Name="ExpandUp">
+                                    <VisualState.Setters>
+                                        <contract7Present:Setter Target="ExpanderHeader.CornerRadius" Value="{Binding CornerRadius, RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource BottomCornerRadiusFilterConverter}}" />
+                                    </VisualState.Setters>
                                     <VisualState.Storyboard>
                                         <Storyboard>
-                                            <PopInThemeAnimation x:Name="PopIn" TargetName="ExpanderContent" FromVerticalOffset="-50" FromHorizontalOffset="0"/>
                                             <ObjectAnimationUsingKeyFrames Storyboard.TargetName="ExpanderContent" Storyboard.TargetProperty="Visibility">
                                                 <DiscreteObjectKeyFrame KeyTime="0" Value="Visible" />
                                             </ObjectAnimationUsingKeyFrames>
+                                            <DoubleAnimationUsingKeyFrames
+                                            Storyboard.TargetName="ExpanderContent"
+                                            Storyboard.TargetProperty="(UIElement.RenderTransform).(CompositeTransform.TranslateY)">
+                                                <DiscreteDoubleKeyFrame KeyTime="0" Value="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.ContentHeight}" />
```

Comment:
>{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.ContentHeight} [](start = 91, length = 94)

Can this be replace by something like: 

```
{Binding RelativeSource={RelativeSource ExpanderContent}, Path=ActualHeight} 
```

This would remove the need for one of the controls template settings, but the negative one would still be required...

Reply:
I think it might be cleaner to keep them both as template settings since we need one anyway and they're mirrored and design might decide we need to move them to actualHeight*2 or whatever other value. This way now it's clear they're coming from the same place as a setting